### DISCUSSION TITLE
How to display the row auto ID after a new row has been submitted?

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#7347] [Help]
Hi Guys,

I wonder if this is possible?

I would like to present the user with the auto id of the row that they have created. 
This would allow me to tell them their application id in case they provide a wrong email address 

I was thinking something like this on a form step:
"Your application number is: {{ autoId }}"

**Comment by melohagan** [comment]
Hi @FlaminWrap 

You can use <a href="https://docs.budibase.com/docs/referencing">action referencing</a> to get the AutoID of the row that was just saved. 
Then you could push this id to <a href="https://docs.budibase.com/docs/app-state">app state</a> and use that binding in your form step paragraph.
